src/test/java/org/springframework/retry/backoff/ExponentialBackOffPolicyTests.java

```diff
@@ -94,6 +94,22 @@ public void testMultiBackOff() {
 		}
 	}
 
+	@Test
+	public void testMultiBackOffWithInitialDelaySupplier() {
+		ExponentialBackOffPolicy strategy = new ExponentialBackOffPolicy();
+		long seed = 40;
+		double multiplier = 1.2;
+		strategy.initialIntervalSupplier(() -> 40L);
+		strategy.setMultiplier(multiplier);
+		strategy.setSleeper(sleeper);
+		BackOffContext context = strategy.start(null);
+		for (int x = 0; x < 5; x++) {
+			strategy.backOff(context);
+			assertThat(sleeper.getLastBackOff()).isEqualTo(seed);
```

Comment:
We need to test that we still have exponential back offs (which we no longer have with the current change).

Reply:
Could you suggest any examaple here?

Reply:
Actually current test checking following, validated in debug.
| Last backoff | Seed |
|--------|--------|
| 40| 40 |
| 48| 48 |
| 57| 57 |
| 68| 68 |
| 81| 81 | 

I was thinking it is sufficient for current issue.